pom.xml

```diff
@@ -134,6 +134,18 @@
             <classifier>mule-plugin</classifier>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mule.runtime</groupId>
+            <artifactId>mule-metadata-model-json</artifactId>
+            <version>1.3.0</version>
+            <scope>provided</scope>
```

Comment:
Does this provided by the application developer?

Reply:
Yes it is, below is the dependency tree : 
 +- org.mule.runtime:mule-metadata-model-json:jar:1.3.0:provided
[INFO] |  +- com.google.code.gson:gson:jar:2.8.5:provided
[INFO] |  +- org.everit.json:org.everit.json.schema:jar:1.5.1:provided
[INFO] |  +- org.mule.runtime:mule-metadata-model-api:jar:1.3.0:provided
[INFO] |  \- org.mule.runtime:api-annotations:jar:1.2.0:provided

Reply:
Still my question is unanswered. Can you please check how these dependency will come in an application. Provided scopes are need to be provided by the user. So in this case application developer. Can you check in anypoint studio. If you do not provide this in application pom what will happen.